docs/topics/strings.md

```diff
@@ -132,6 +136,9 @@ fun main() {
     val s = "abc"
     println("$s.length is ${s.length}") 
     // abc.length is 3
+    val numbers = listOf(1,2,3,4,5)
+    println("Last two numbers: ${numbers.takeLast(2)}")
+    // Last two numbers: [4, 5]
```

Comment:
I think this addition is redundant. We already made a point about `toString` in the previous example

Reply:
I agree, that's redundant. Thanks!